src/foam/demos/u2/MDAllViews.js

```diff
@@ -322,7 +322,7 @@ foam.CLASS({
       class: 'FObjectArray',
       name: 'fobjectArray',
       of: 'foam.demos.u2.SampleData',
-      view: { class: 'foam.u2.view.FObjectArrayView', valueView: 'foam.demos.heroes.CitationView' }
+      view: { class: 'foam.u2.view.TitledArrayView', valueView: 'foam.demos.heroes.CitationView' }
```

Comment:
recomend the same thing as above - use a multiView for both - for all demo/AllView